examples/flutter_gallery/test_driver/transitions_perf.dart

```diff
@@ -18,5 +19,8 @@ Future<String> _handleMessages(String message) async {
 
 void main() {
   enableFlutterDriverExtension(handler: _handleMessages);
-  app.main();
+  // As in lib/main.dart: overriding https://github.com/flutter/flutter/issues/13736
+  // for better visual effect at the cost of performance.
```

Comment:
But if this is a perf test, why would we want to do that?  Does it give more consistent results or something?

Reply:
The performance test baselines already account for this slowdown, so leaving it in will keep the results consistent.